build.gradle

```diff
@@ -19,6 +19,7 @@ buildscript {
 
 allprojects {
     repositories {
+        jcenter()
```

Comment:
Why did u add jcenter().

Reply:
Because the build was failing and it was showing 7 libraries as failed to resolve.

Reply:
@atm1504  can you please re-trigger the build or help me identify the reason for build fail error?